doc/api/v8.md

```diff
@@ -37,7 +37,7 @@ changes:
 Returns statistics about the V8 heap spaces, i.e. the segments which make up
 the V8 heap. Neither the ordering of heap spaces, nor the availability of a
 heap space can be guaranteed as the statistics are provided via the V8
-[`GetHeapSpaceStatistics`][] function and may change from one V8 version to the
+[`GetHeapSpaceStatistics`](https://v8docs.nodesource.com/node-8.11/d2/d03/classv8_1_1_heap_space_statistics.html) function and may change from one V8 version to the
```

Comment:
The link is currently defined on L420 and should be updated there instead of inline here.
```markdown
[`GetHeapSpaceStatistics`]: https://v8docs.nodesource.com/node-8.9/d5/dda/classv8_1_1_isolate.html#ac673576f24fdc7a33378f8f57e1d13a4
```

The current link 404's, so we could either replace it with the equivalent Node 8:
https://v8docs.nodesource.com/node-8.11/d5/dda/classv8_1_1_isolate.html#ac673576f24fdc7a33378f8f57e1d13a4

or Node 10, which is closer to what is in `master`:
https://v8docs.nodesource.com/node-10.6/d5/dda/classv8_1_1_isolate.html#ac673576f24fdc7a33378f8f57e1d13a4

Reply:
Looking at https://v8docs.nodesource.com/ and the available options, I think this link is likely to break often when we put out new releases.

Reply:
Agreed. We are going to need something more automated to keep these in sync. For now, updated this PR with your suggestion to move it to L420 and use the Node10 link. It may look odd in Node8 docs but that function hasn't changed much.